### PR TITLE
Restructure and docker

### PR DESCRIPTION
**Pull request by Autom3**
Full restructure of the python project according to the PEP conventions.

For local runs

- install [poetry](https://python-poetry.org/docs/#installation)
- run `poetry install`
- run `python nerdlandbot`

For docker runs:

- clone repo
- cd to repo
- run `docker build -t nerdlandbot:latest .`
- run this command (which filled in variables):
```
docker run -itd --restart="unless-stopped" --name nerdlandbot \
 -e PREFIX=<Your prefix here> \
 -e DISCORD_TOKEN=<Your discord token here> \
 -v /bind/mount/path/for/guild/configs:/GuildConfigs \
 nerdlandbot
```

### CODE REVIEW

**Review by DrSkunk — APPROVED** [review]
🚢

**Review by DrSkunk — COMMENTED** [review]
Add info on using Poetry on Readme, the rest is OK.

**Review by DrSkunk — CHANGES_REQUESTED** [review]
Add info on using Poetry on Readme, the rest is OK.

Not as a comment this time

**Review by bobvdvleuten — COMMENTED** [review]
shouldn't this go to develop branch and then to master instead of straight to master?

File: pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+[tool.poetry]
+name = "nerdlandbot"
+version = "0.1.0"
+description = ""
+authors = ["Autom"]
```
Comment (bobvdvleuten) [inline pyproject.toml]: nitpicking, should be more than only Autom, right ;)

Comment (Autom3) [reply pyproject.toml]: It definitely should be, but I don't know what nickname everyone wants there...

Comment (bobvdvleuten) [reply pyproject.toml]: Would suggest to make it their github usernames?

Comment (Autom3) [reply pyproject.toml]: Ah I used the names people used to commit to git, that was a bit easier for me :P People can always change theirs in a PR if they want?

**Comment by Autom3** [comment]
@DrSkunk Are the changes to README.md sufficient in terms of poetry instructions? If not, how would you suggest I change it?

**Pull request closed without merging**